crates/bevy_render/src/experimental/occlusion_culling/mesh_preprocess_types.wgsl

```diff
@@ -50,8 +50,13 @@ struct IndirectParametersMetadata {
     mesh_index: u32,
     base_output_index: u32,
     batch_set_index: u32,
+#ifdef WRITE_INDIRECT_PARAMETERS_METADATA
     early_instance_count: atomic<u32>,
     late_instance_count: atomic<u32>,
+#else
+    early_instance_count: u32,
+    late_instance_count: u32,
+#endif
```

Comment:
Nice, I had wanted to make this small performance improvement for a while but you went ahead and did it, thanks!

Reply:
It's actually a bug that had to be fixed, chrome with WebGPU throws an error if you have atomic variables in a read-only buffer.